### ISSUE TITLE
[dev.icinga.com #12204] Improve author information about check_yum

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icinga2#4387]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/12204

**Created by calestyo on 2016-07-22 01:43:32 +00:00**

Assignee: _mfriedrich_
Status: _Resolved_  (closed on _2016-07-25 15:35:06 +00:00_)
Target Version: _2.5.0_
Last Update: _2016-07-25 15:35:06 +00:00 (in Redmine)_

    Icinga Version: 2.4.10
    Backport?: Not yet backported
    Include in Changelog: 1

---

Hey.

At
http://docs.icinga.org/icinga2/latest/doc/module/icinga2/toc\#!/icinga2/latest/doc/module/icinga2/chapter/plugin-check-commands#plugin-contrib-command-yum
and perhaps other locations, you name myself (Christoph Anton Mitterer) as author from check\_yum.

It's correct that I'm still maintaining it (more or less - probably rather "less" than "more"), but I've only took that over because it was completely abandoned at some time and there were many different locations with different patches.

Long story short, most of the code is still from the original author, and I don't think I should be credited so much there. Just link to the repo and that's fine, everyone who is interested in the contributors can look up the git log :-)

Best wishes,
Chris.

**Changesets**

_2016-07-25 15:33:19 +00:00_ by _mfriedrich_ f2bacb26970324fd7fef1d70d05f768549f6f59b

    Docs: Improve yum CheckCommand
    
    fixes #12204
